src/deploy.ts

```diff
@@ -330,7 +330,12 @@ export async function deploy(
         );
       } else {
         clack.log.error(
-          wrapAnsi(`Could not create project: ${error instanceof Error ? error.message : error}`, effects.outputColumns)
+          wrapAnsi(
+            `Could not create project: ${
+              (error as any)?.statusCode === 409 ? "conflicting slug." : error instanceof Error ? error.message : error
```

Comment:
The type safe version of this would be something like 

```ts
if (isErrorMessage(error) && error.statusCode === 409 ...
```

This is probably fine as is.